### PR TITLE
Enable / in reference search

### PR DESCRIPTION
**Pull request by Ben0189**
Solve bug in search #4763

### CODE REVIEW

File: packages/tinacms/src/toolkit/fields/components/reference/reference-select.tsx [hunk 1]
```diff
@@ -177,13 +177,7 @@ const ComboboxDemo: React.FC<ReferenceSelectProps> = ({
             shouldFilter={!field.experimental___filter}
             filter={(value, search) => {
               //Replace / in the file path with empty string to make it searchable
```
Comment (ncn-ssw) [inline packages/tinacms/src/toolkit/fields/components/reference/reference-select.tsx:179]: **question**: Is this comment still relevant?

File: .changeset/modern-cougars-clap.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'tinacms': patch
+---
+
+enable / in search
```
Comment (kldavis4) [inline .changeset/modern-cougars-clap.md:5]: Can you provide more details for the change set?

Comment (Ben0189) [reply .changeset/modern-cougars-clap.md]: Done - updated changeset

File: packages/tinacms/src/toolkit/fields/components/reference/reference-select.tsx (hunk 1 above)
Comment (Ben0189) [reply packages/tinacms/src/toolkit/fields/components/reference/reference-select.tsx]: Done - removed redundant comment

**Review by Jord-Gui — APPROVED** [review]

**Pull request merged**